### DISCUSSION TITLE
Search within project

### DISCUSSION THREAD

**Discussion by koengrosman** [giovannicoppola/AlfreDo#21] [Q&A]
Hi!,

Thanks for the great workflow.

Is it also possible to not search for a specific task, but to have an overview of all the open tasks within a certain project.

So for example I have a project "House", that I search on #house for example and see the 4 open tasks within that project?

Maybe that is already possible, but I couldn't find it. Thanks in advance!

**Comment by giovannicoppola** [comment]
thanks for pointing this out @koengrosman! This feature was there but was lost in the last version update. Please check [this version](https://github.com/giovannicoppola/AlfreDo/releases/tag/v0.4.1) and let me know if you see any problems.

**Comment by koengrosman** [comment]
Great, thank you :) It works now!

Op ma 20 mei 2024 om 11:23 schreef Giovanni ***@***.***>:

**Comment by koengrosman** [comment]
Hi Giovanni,

One more question.

Is it also possible to alter an existing task with AlfredDo? Or to add a
comment to a certain task within AlfredDo?<-- My preference

Just asking :) And have a great day.

Op ma 20 mei 2024 om 11:38 schreef Koen Grosman ***@***.***>:

**Reply by giovannicoppola** [reply]
not at the moment (other than rescheduling), but it is a good idea and I added it as a possible future enhancement.

**Comment by koengrosman** [comment]
Thanks for the update :)

Op ma 20 mei 2024 om 12:35 schreef Giovanni ***@***.***>

**Comment by koengrosman** [comment]
For the reply i meant

Op ma 20 mei 2024 om 13:56 schreef Koen Grosman ***@***.***>

**Comment by giovannicoppola** [comment]
@koengrosman this is implemented in version 0.5, please feel free to test the pre-release here: [version 0.5](https://github.com/giovannicoppola/AlfreDo/blob/dev/releases/AlfreDo_0.5.alfredworkflow), and let me know if something does not work!
